### DISCUSSION TITLE
ONVIF device test tool for consumers?

### DISCUSSION THREAD

**Discussion by mjmeans** [onvif/testspecs#80] [Q&A] [answered]
I got a PTZ camera from a Chinese manufacturer that I think is not compliant. The PTZ features do not work with frigate NVR using the default port 8000. An ONVIF device manager reports that the device ports are 8080 for the web interface but that port also does not accept PTZ commands. I have a suspicion that this device doesn't actually pass ONVIF specs. How can I, as a consumer, verify that it doesn't pass so that I can post a bad Amazon review and provide the evidence to back it up?

**Comment by madhurao68** [comment, marked as answer]
Hi,
The best way to verify is to check if the specific device is listed on the ONVIF Conformant Products page (https://www.onvif.org/conformant-products/).  

![image](https://github.com/onvif/testspecs/assets/73811069/a64f6587-e166-40c1-a129-815224147dae)



Hope this helps!
